### DISCUSSION TITLE
insufficient funds for gas * price + value

### DISCUSSION THREAD

**Discussion by chioke07** [smartcontractkit/full-blockchain-solidity-course-py#1858] [Q&A]
<img width="1512" alt="Screenshot 2023-04-03 at 11 42 18 PM" src="https://user-images.githubusercontent.com/129320402/229612451-63ae1f2f-4f9b-492b-ac3a-734ab11799e5.png">

please what could be the cause of this error and how can i fix it? i'm using goerli network because rinkeby is no longer available on metamask testnet. i have 0.02 GoerliETH and 20 LINKS do i need to add more?

@cromewar 

brownie run scripts/1_deploy_token.py --network goerli
Brownie v1.19.3 - Python development framework for Ethereum

Erc20BrownieProject is the active project.

Running 'scripts/1_deploy_token.py::main'...
  File "brownie/_cli/run.py", line 51, in main
    return_value, frame = run(
  File "brownie/project/scripts.py", line 110, in run
    return_value = f_locals[method_name](*args, **kwargs)
  File "./scripts/1_deploy_token.py", line 10, in main
    our_token = OurToken.deploy(initial_supply, {"from": account})
  File "brownie/network/contract.py", line 549, in __call__
    return tx["from"].deploy(
  File "brownie/network/account.py", line 510, in deploy
    receipt, exc = self._make_transaction(
  File "brownie/network/account.py", line 752, in _make_transaction
    exc = VirtualMachineError(e)
  File "brownie/exceptions.py", line 96, in __init__
    raise ValueError(exc["message"]) from None
ValueError: insufficient funds for gas * price + value

**Comment by cromewar** [comment]
Be sure to have enough eth to deploy this on goerli, also please share the deployment code on python here.

**Reply by Aishwaria8786** [reply]
I am facing the similar problem.
